tests/unit/fixtures/zowe.config.user.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Please add some line and block comments in either this file or the `zowe.config.json` fixture. The contents of the comments don't matter, they'll just help to verify that ProfileManager is able to load JSON with comments.